### PR TITLE
SCP-2545 - Marlowe Run Client - Implement transaction summary

### PR DESCRIPTION
**Pull request by hrajchert**
This PR adds a tooltip to the past actions card with the timezone information and includes an initial version of the transaction/payment summary.

time-tooltip

payment_summary

While I was implementing this feature, I discovered a bug in the balance tab of the simple escrow example. When the buyer deposits and chooses to report a problem, there is a payment from the Sellers account to the Buyers account (which is shown by the new payment summary). But in the balances tab the money is removed from the Sellers account but is not added to the Buyers.

Pre-submit checklist:
- Branch
    - [X] Commit sequence broadly makes sense
    - [X] Key commits have useful messages
    - [X] Relevant tickets are mentioned in commit messages
    - [X] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [X] Self-reviewed the diff
    - [X] Useful pull request description
    - [X] Reviewer requested

### CODE REVIEW

File: marlowe-dashboard-client/src/Humanize.purs [hunk 1]
```diff
@@ -93,6 +95,34 @@ getTimezoneOffset = wrap <<< negate <<< unwrap <$> Now.getTimezoneOffset
...
+  where
+  min' = floor $ abs min
+
+  hours = min' / 60
+
+  offsetString =
+    if min' `mod` 60 == zero then
+      show hours
+    else
+      show hours <> ":" <> formatMinutes (min' - hours * 60)
```
Comment (palas) [inline marlowe-dashboard-client/src/Humanize.purs:124]: Nice, it even considers minute offsets, cool :)

**Review by palas — APPROVED** [review]
Nice! I am a bit troubled by `to` being to the left of `from`, not gonna lie, but other than that :)

**Comment by palas** [comment]
Oh, I see now `IDeposit` is that way around too. Anyway, I have changed that as I was fixing purty

**Comment by hrajchert** [comment]
Yes, i dont like the order either, but i tried to be consistent with Deposit

**Pull request merged**